lib/FusionInventory/Agent/Task/Inventory/Win32/License.pm

```diff
@@ -43,7 +43,7 @@ sub doInventory {
         push @licenses, _scanOfficeLicences($officeKey32) if $officeKey32;
     }
 
-    push @licenses, _getWmiSoftwareLicensingProducts();
+    @licenses = _getWmiSoftwareLicensingProducts(@licenses);
```

Comment:
It would be more efficient here to just pass the array reference. You then don't need to retrieve anything from the method.